src/app/helpers/embedGoogleMaps/embedGoogleMaps.ts

```diff
@@ -27,9 +27,9 @@ export async function embedGoogleMaps(key?: string) {
     function mapLoaded() {
       if (typeof google !== "undefined") {
         resolve();
-      } else if (count > 5) {
+      } else if (count > 10) {
         console.error("Failed to load google maps.");
-        reject();
+        reject("Google Maps API Bundle took too long to download.");
```

Comment:
"try refreshing the page"?

Reply:
Official error message already does that, that error message will be printed to the console for developers.